package.json

```diff
@@ -2802,7 +2802,7 @@
                   ]
                 },
                 "statusBarLength": {
-                  "description": "%cmake-tools.configuration.cmake.options.advanced.statusBarLength.description%",
+                  "description": "%cmake-tools.configuration.cmake.options.advanced.statusBarLength.markdownDescription%",
```

Comment:
I believe there are 12 locations in `package.json` that need to be updated because this string resource is resused.